client/src/components/byline-postpicker/index.jsx

```diff
@@ -0,0 +1,48 @@
+// External dependencies.
+import apiFetch from '@wordpress/api-fetch';
+import { addQueryArgs } from '@wordpress/url';
+import { useCallback } from '@wordpress/element';
```

Comment:
WP has moved away from importing from `@wordpress/element` to importing from `react` and we should do the same here:

```suggestion
import { useCallback } from 'react';
```